gpudrive/datatypes/observation.py

```diff
@@ -406,8 +406,11 @@ def __init__(
         traj_length = constants.LOG_TRAJECTORY_LENGTH
         self.tl_states = tl_states_tensor[:, :, 1:traj_length]
         self.lane_id = tl_states_tensor[:, :, 0]
-        self.tl_xyz = tl_states_tensor[:, :, traj_length:traj_length + 3]
-        self.time_index = tl_states_tensor[:, :, traj_length + 3:traj_length + 4]
+        # self.tl_xyz = tl_states_tensor[:, :, traj_length:traj_length + 3]
+        self.pos_x = tl_states_tensor[:, :, traj_length:2*traj_length - 1]
```

Comment:
Could we change the export in types.hpp such that the shape of the `tl_state_tensor` is `(num_worlds, num_tls, num_featurs, 91)` ? Currently the last two dimensions are flattened, which makes the code hard to read.